HIPS/hip-1.md

```diff
@@ -0,0 +1,54 @@
+---
+hip: <to be assigned>
+title: <HIP title>
+author: <a list of the author's or authors' name(s) and/or username(s), or name(s) and email(s), e.g. (use with the parentheses or triangular brackets): FirstName LastName (@GitHubUsername), FirstName LastName <foo@bar.com>, FirstName (@GitHubUsername) and GitHubUsername (@GitHubUsername)>
+status: Example
+type: <Standard Track | Informational>
+category (*only required for Standard Track): <Core | Interface | Economics | HRC>
```

Comment:
The idea behind HRC was mostly that there are decisions that can (and should) be going through the HIP process that doesn't require Hedera involvement. For instance, any community-supported tools. Additionally, HRC is an explicit way of flagging that you need the team's input before something can move forward. Happy to remove if we don't feel there's value there, however.

Reply:
So I guess the HRC description is just too vague in the README and. The EIP defines ERC as:
`application-level standards and conventions, including contract standards such as token standards (ERC20), name registries (ERC26, ERC137), URI schemes (ERC67), library/package formats (EIP82), and wallet formats (EIP75, EIP85).`

This makes more sense, though the name is more obtuse. I would suggest maybe calling it `Application` or `Tools` instead of HRC and updating the readme would help clarify things?

Reply:
Do you have a suggested delineation? I'm not clear on how product expects to manage standards, conventions, things like that, and what role we will or won't play vs the community and so I would not be sure what distinctions are valuable.

Reply:
I'm not entirely sure, though I'm not sure of any of this process because we haven't documented much of the process. It could be that informational and Application HIPs go through a different process that is entirely community approved.

Reply:
I agree that there could be informational and HIPs that go through a process that is not described adequately in the existing README. Perhaps a simple fix would be making steps 5 & 6 in the "workflow" section flagged as (if applicable)? Open to other suggestions.

Reply:
Sounds good, except instead of if applicable be specific and say 5 & 6 applies only to standard HIP.

Reply:
Added in b123da7917e6021be183edc95c234e2de98da1c8 but feel free to let me know if we should reword.